### PR TITLE
add(tools): Inspect Flow

### PR DESCRIPTION
**Pull request by GuillermoCoding**
| Name                 | Link                 |
| -------------------- | -------------------- |
| _Inspect Flow_ | _inspectflow.io_ |

_Inspect Flow is a browser extension to quickly inspect, edit, and copy the Tailwind CSS classes of elements!._

---

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] My item is in line with the [Tailwind brand usage guidelines](https://tailwindcss.com/brand#usage)
- [x] I have read and followed the [contribution guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -228,6 +228,7 @@
 - 🌍 [Tailwind Cheat Sheet](https://github.com/LeCoupa/awesome-cheatsheets/blob/master/frontend/tailwind.css) - Tailwind CSS class names in one single file.
 - 🌍 [Tailwind Cheat Sheet](https://umeshmk.github.io/Tailwindcss-cheatsheet) - Tailwind CSS class names, variants and directives cheat sheet.
 - 🌍 [Tailwind Cheat Sheet](https://tailwindcomponents.com/cheatsheet) - Tailwind CSS class names in a searchable page.
+- 💼🌍 [Inspect Flow](https://inspectflow.io) - The complete developer tool for Tailwind CSS.
```
Comment (joshmanders) [inline README.md:231]: I feel like this could be up under `clb`, but I'm not sure. What are your thoughts @innocenzi?

Comment (innocenzi) [reply README.md:231]: I agree with that 👍 Feels better being grouped there to me

Comment (joshmanders) [reply README.md:231]: @GuillermoCoding if you can make this change, we'll merge you in!

Comment (GuillermoCoding) [reply README.md:231]: Awesome, going to make the change. Thanks @joshmanders @innocenzi !

**Review by joshmanders — APPROVED** [review]
Thanks!

**Pull request merged**